multiple_buyers/entry.json

```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Multiple_Buyers_Extension"
+  },
+  "description": {
+    "en": "This extension provides a way to provide multiple buyers per contract."
```

Comment:
I would suggest updating the description to:

"Adds a buyers array to each contract, allowing one or more buyers to be specified for each individual contract within an overall contracting process."